docs/getting_started/permissions.md

```diff
@@ -1,15 +1,41 @@
 ## Permissions
 
-<!-- TODO(lucacasonato): what are permissions -->
+Deno is secure by default. Therefore, unless you specifically enable it, a deno
+module has no file, network, or environment access for example. Access to
+security sensitive areas or functions requires the use of permissions to be
+granted to a deno process on the command line.
 
-<!-- TODO(lucacasonato): description of all permissions -->
+For the following example, `mod.ts` has been granted read-only access to the
+file system. It cannot write to it, or perform any other security sensitive
+functions.
+
+```shell
+deno run --allow-read mod.ts
+```
+
+### Permissions list
+
+The following permissions are available:
+
+| Command&nbsp;line&nbsp;flag&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Permission description                                                                                                                                                                     |
+| --------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
```

Comment:
Tables look really nice when rendered, but they're difficult to maintain. It would be better if you used a list here.